code/game/gamemodes/blob/overmind.dm

```diff
@@ -91,10 +91,14 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 
 		if(!Ablob.blob_allowed)
 			continue
-
-		playsound(L, 'sound/effects/splat.ogg', 50, 1)
-		L.death()
-		new/mob/living/simple_animal/hostile/blob/blobspore(T)
+		
+		if(!istype(L, /mob/living/simple_animal/hostile/blob))
+			playsound(L, 'sound/effects/splat.ogg', 50, 1)
+			L.death()
+			new/mob/living/simple_animal/hostile/blob/blobspore(T)
+		else
+			L.maxHealth = INFINITY
+			L.health = INFINITY
```

Comment:
Forget this shit

Reply:
Why? I mean the round is over by this point, but i'll change it to just a full heal.